packages/usa-nav/src/styles/_usa-nav.scss

```diff
@@ -161,15 +162,23 @@ $expand-less-icon: map-merge(
     > a {
       @include at-media($theme-header-min-width) {
         @include primary-nav-link;
+        align-items: center;
         color: color($nav-link-color);
-        display: block;
+        display: flex;
         font-weight: font-weight("bold");
 
         &:hover {
           color: color("primary");
         }
       }
     }
+
+    > button,
+    > a {
+      @include at-media($theme-header-min-width) {
+        height: 100%;
+      }
```

Comment:
> [!CAUTION]
> Selectors like this are heavy handed and could cause breaking changes. Acceptable because this component heavily relies on this pattern. We should use this with a lot of caution.